<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>137</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="insert_pushButton">
     <property name="minimumSize">
      <size>
       <width>115</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>115</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>上架商品</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="updata_pushButton">
     <property name="minimumSize">
      <size>
       <width>115</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>115</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>修改商品信息</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="delete_pushButton">
     <property name="minimumSize">
      <size>
       <width>115</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>115</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>下架商品</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
